src/components/views/elements/ImageView.tsx

```diff
@@ -360,16 +407,17 @@ export default class ImageView extends React.Component<IProps, IState> {
         const showEventMeta = !!this.props.mxEvent;
         const zoomingDisabled = this.state.maxZoom === this.state.minZoom;
 
+        let transition;
+        if (this.animatingLoading) transition = "transform 300ms ease 0s";
```

Comment:
That seems like an easy way to get race conditions considering the above code, could you triple check if it works as intended even if you slow down the CPU in chrome devtools?

Reply:
Seems to work fine :tada: